CHANGELOG.md

```diff
@@ -287,7 +295,8 @@ use `Band.create`
 
 Initial release.
 
-[Unreleased]: <https://github.com/stac-utils/pystac/compare/v0.5.6...main>
+[Unreleased]: <https://github.com/stac-utils/pystac/compare/v1.0.0-beta.1...main>
+[v1.0.0-beta.1]: <https://github.com/stac-utils/pystac/compare/v0.5.6...v1.0.0-beta.1>
```

Comment:
Looks like these links are inconsistent across versions. Some use `...` (3 dots), which just lists the commits, and some use `..` which shows the file diffs. My preference would be for all of them to use `..` (2 dots) and show the file diffs, but either way we should make them consistent throughout.

```suggestion
[Unreleased]: <https://github.com/stac-utils/pystac/compare/v1.0.0-beta.1..main>
[v1.0.0-beta.1]: <https://github.com/stac-utils/pystac/compare/v0.5.6..v1.0.0-beta.1>
```

Also the `## unreleased` header should probably be changed to `## [Unreleased]` so it matches the link here.

Reply:
Good call, will make that change. Thanks!

Reply:
> Looks like these links are inconsistent across versions. Some use `...` (3 dots), which just lists the commits, and some use `..` which shows the file diffs. My preference would be for all of them to use `..` (2 dots) and show the file diffs, but either way we should make them consistent throughout.
> 
> Also the `## unreleased` header should probably be changed to `## [Unreleased]` so it matches the link here.

Actually, scratch that. Now that I've seen how many example files change between versions I think we should go with `...` and just show the list of commits 😄

Reply:
👍